cbdr/cortex
# Database Config Simplification

**Purpose:**
* Simplifies `database.yml` config & environmental variables
* Remove deprecated config option so C66 db mirroring works
  * The error in question can be seen here: https://app.cloud66.com/stacks/33681/managed_backups/9167911/restore_progress

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-912

**Changes:**
* Changes to setup
  * No longer need as many environmental variables setup for `test` environment!

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
N/A

* After
N/A

**QA Links:**
N/A

**How to Verify These Changes**
* Specific pages to visit
  * N/A

* Steps to take
  * Ensure application still runs locally

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies:**
N/A

**Additional Information**
N/A
